c2corg_ui/templates/article/filters.html

```diff
@@ -0,0 +1,63 @@
+<%!
+from c2corg_common.attributes import default_langs, quality_types, \
+activities, article_categories, article_types
+%>
+<%namespace file="../helpers/filters.html" import="add_multiselect"/>
+<form app-search-filters app-search-filters-controller-name="appSearchFiltersController">
+  <div class="filters">
+    <div class="row simple-filters">
+
+      <div class="filter">
+        <label translate>title</label>
+        <input type="text" class="form-control" ng-model="filtersCtrl.filters.q">
+      </div>
+
+      <div class="filter">
+        <label translate>Activities</label><br>
+        ${add_multiselect('act', activities)}
+      </div>
+
+      <div class="filter">
+        <label translate>Article categories</label><br>
```

Comment:
Perhaps simply ``categories``? (already exist in the translated strings, in Transifex).
Or ``article_categories`` if this attribute name will be added to the strings to translate.